### PR TITLE
Add 6.2.5 release notes

### PR DESCRIPTION
**Pull request by cwjohnston**
## Description

Add release notes for 6.2.5 patch release in 6.2 and 6.3 content.

## Motivation and Context

Support patch release.

### CODE REVIEW

**Review by echlebek — APPROVED** [review]
:shipit:

File: content/sensu-go/6.2/release-notes.md [hunk 1]
```diff
@@ -79,6 +80,18 @@ Read the [upgrade guide][1] for information about upgrading to the latest versio
 
 ---
 
+## 6.2.5 release notes
+
+**February 2, 2021** &mdash; The latest release of Sensu Go, version 6.2.5, is now available for download.
+
+This patch fixes a bug regarding event occurrence watermark property. This bug interfered with expected behavior when using event filters like the popular fatigue check filter.
```
Comment (hillaryfraley) [inline content/sensu-go/6.2/release-notes.md]:
```suggestion
This patch fixes a bug regarding the event occurrences_watermark property.
This bug interfered with the property's expected behavior when using event filters like the popular fatigue check filter.
```

File: content/sensu-go/6.2/release-notes.md [hunk 2]
```diff
@@ -79,6 +80,18 @@ Read the [upgrade guide][1] for information about upgrading to the latest versio
...
+
+See the [upgrade guide][1] to upgrade Sensu to version 6.2.5.
+
+**FIXES:**
+
+- ([Commercial feature][193]) Fixed a bug where occurrences_watermark wouldn't increment for non-zero events when using the PostgreSQL datastore.
```
Comment (hillaryfraley) [inline content/sensu-go/6.2/release-notes.md]:
```suggestion
- ([Commercial feature][193]) Fixed a bug that prevented occurrences_watermark from incrementing for non-zero events when using the PostgreSQL datastore.
```

**Review by hillaryfraley — APPROVED** [review]
Just a couple suggestions

**Pull request merged**